<launch>

        <arg name="search_alt_" default="0.5" />
        <arg name="markers_id_" default="0" />
        <arg name="desire_pose_x" default="0" />
        <arg name="desire_pose_y" default="0" />
        <arg name="desire_pose_z" default="1" />
        <arg name="desire_yaw_" default="0" />
        <arg name="PidXY_p" default="0.3" />
        <arg name="PidXY_d" default="0.05" />
        <arg name="PidXY_i" default="0.01" />
        <arg name="PidZ_p" default="0.12" />
        <arg name="PidZ_d" default="0" />
        <arg name="PidZ_i" default="0.01" />
        <arg name="PidYaw_p" default="0" />
        <arg name="PidYaw_d" default="0" />
        <arg name="PidYaw_i" default="0" />
        <arg name="big_marker_size" default="17.5" />
        <arg name="small_marker_size" default="6.9" />	
		<arg name="Thres_count_detect" default="15" />
		<arg name="Thres_count_approach" default="15" />
		<arg name="Thres_count_small" default="20" />
		<arg name="Thres_count_land" default="5" /> <!-- 5 --> 
		<arg name="landing_vel" default="-0.1" />

	<node pkg="uavros_artaglanding" type="landing_control_p200_node" name="landing_control_p200_node" output="screen" >
		<param name="search_alt_" value = "$(arg search_alt_)"/>	
		<param name="markers_id_" value = "$(arg markers_id_)"/>
		<param name="desire_pose_x" value = "$(arg desire_pose_x)"/>	
		<param name="desire_pose_y" value = "$(arg desire_pose_y)"/>	
		<param name="desire_pose_z" value = "$(arg desire_pose_z)"/>
		<param name="desire_yaw_" value = "$(arg desire_yaw_)"/>	
		<param name="PidXY_p" value = "$(arg PidXY_p)"/>	
		<param name="PidXY_d" value = "$(arg PidXY_d)"/>	
		<param name="PidXY_i" value = "$(arg PidXY_i)"/>	
		<param name="PidZ_p" value = "$(arg PidZ_p)"/>	
		<param name="PidZ_d" value = "$(arg PidZ_d)"/>	
		<param name="PidZ_i" value = "$(arg PidZ_i)"/>	
		<param name="PidYaw_p" value = "$(arg PidYaw_p)"/>	
		<param name="PidYaw_d" value = "$(arg PidYaw_d)"/>	
		<param name="PidYaw_i" value = "$(arg PidYaw_i)"/>
		<param name="big_marker_size" value = "$(arg big_marker_size)"/>
		<param name="small_marker_size" value = "$(arg small_marker_size)"/>
		<param name="Thres_count_detect" value = "$(arg Thres_count_detect)"/>
		<param name="Thres_count_approach" value = "$(arg Thres_count_approach)"/>	
		<param name="Thres_count_small" value = "$(arg Thres_count_small)"/>	
		<param name="Thres_count_land" value = "$(arg Thres_count_land)"/>	
		<param name="landing_vel" value = "$(arg landing_vel)"/>		
			
	</node>
</launch>
